Neos.Media.Browser/Resources/Public/Styles/MediaBrowser.css

```diff
@@ -26,11 +26,11 @@
   src: local("Noto Sans Bold Italic"), local("NotoSans-BoldItalic"), url("../Fonts/NotoSans/NotoSans-BoldItalic.ttf") format("truetype");
 }
 /*!
- * Font Awesome Free 5.8.2 by @fontawesome - https://fontawesome.com
+ * Font Awesome Free 5.12.1 by @fontawesome - https://fontawesome.com
  * License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
  */
 @font-face {
-  font-family: 'FontAwesome';
+  font-family: 'Font Awesome 5 Free';
```

Comment:
This change is unrelated to the PR, but seems that the pipeline for the Media-Browser did not run for some time.